{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/tasks.py

```diff
@@ -1,12 +1,14 @@
 {%- if cookiecutter.use_celery == 'y' -%}
 from celery.utils.log import get_task_logger
+import structlog
```

Comment:
:P

not sure I assumed ruff will fail CI here, so but I guess it is not enabled by default so it doesnt